jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
```

Comment:
I do not consider a 400 line bash file trivial, and I would like to see the complexity of our bash files approach zero over time.

This PR seems to be moving in the other direction: nearly 100 lines of new bash code, lots of branches, state is maintained in global environment variables.